runtime/runtime_test.go

```diff
@@ -369,6 +369,72 @@ func TestServerInitialized(t *testing.T) {
 	}
 }
 
+func TestWarningMessage(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Millisecond)
+	defer cancel() // NOTE(sr): The timeout will have been reached by the time `done` is closed.
+
+	params := NewParams()
+
+	logger := logging.New()
```

Comment:
I think there might be a `TestLogger()` somewhere in the code base 🔍

Reply:
💡 https://github.com/open-policy-agent/opa/blob/afabf5a5047993ec02d6d87e7585afb20aa282f6/logging/test/test.go#L26

Reply:
Awesome! Will take a look! :+1:

Reply:
I made some changes, I **think** I understood what you wanted me to do, but if I missed something please let me know :sweat_smile: either way I think the tests are slightly more thorough now :+1: